src/containers/StructurePage/StructureContainer.jsx

```diff
@@ -399,17 +439,32 @@ export class StructureContainer extends React.PureComponent {
                 />
               )
             }
+            toggleSwitch={
+              <Switch
+                onChange={() => this.toggleShowFavorites()}
```

Comment:
Her trenger vi ikke lage en ny funksjon, vi kan bare sende inn den som finnes.
```suggestion
                onChange={this.toggleShowFavorites}
```